components/widgets/resources/src/main/resources/resources/uicomponents/widgets/extraGeneVariantData.js

```diff
@@ -427,6 +439,14 @@ var ExtraGeneVariantData = (function (ExtraGeneVariantData) {
       });
     },
 
+    restrictNumericInput : function () {
+      this._geneTable.select('input:[id$=_start_position]', 'input:[id$=_end_position]').each( function(input) {
+        input.observe('keyup', function(event){
+          this.value=this.value.replace(/[^\d]/,'');
```

Comment:
`keyup` is not enough, since our users are likely to paste with the mouse. The [`input` event](https://developer.mozilla.org/en-US/docs/Web/Events/input) seems to be supported by our target browsers, and would cover any kind of change.